packages/api/src/routes/sdkRoutes.ts

```diff
@@ -61,6 +61,51 @@ export const sdkRoutesWrapper = (storeManager: Promise<dbManager.DataStoreManage
         }
     });
 
+    sdkRoutes.get("/api/sdk/rulesbyflag", auth.isAuthenticated, async (req: Request, res: Response) => {
```

Comment:
to keep consitency you can call the url:` /flags/:flagKey/rules` and read the data from the` req.params`